### ISSUE TITLE
[Fabric 1.21] Crash on launch

### ISSUE THREAD

**Issue by Poopooracoocoo** [Scouter456/Nether_Depths_Upgrade#37]
## Crash report

[crash-2024-06-22_21.49.35-client.txt](https://github.com/user-attachments/files/15936527/crash-2024-06-22_21.49.35-client.txt)

<details><summary>Click to expand crash report</summary>

```
---- Minecraft Crash Report ----
// Hi. I'm Minecraft, and I'm a crashaholic.

Time: 2024-06-22 21:49:35
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'netherdepthsupgrade'!
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:388)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:386)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:477)
	at net.minecraft.client.main.Main.main(Main.java:239)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
Caused by: java.lang.IncompatibleClassChangeError: class com.scouter.netherdepthsupgrade.entity.LavaAnimal overrides final method net.minecraft.class_1309.method_6094()Z
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1027)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:160)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:355)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1027)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:160)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:355)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1027)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:160)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:355)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at com.scouter.netherdepthsupgrade.entity.NDUEntity.<clinit>(NDUEntity.java:18)
	at com.scouter.netherdepthsupgrade.items.NDUItems.<clinit>(NDUItems.java:86)
	at com.scouter.netherdepthsupgrade.setup.Registration.init(Registration.java:25)
	at com.scouter.netherdepthsupgrade.NetherDepthsUpgrade.onInitialize(NetherDepthsUpgrade.java:31)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:384)
	... 9 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:388)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:386)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:477)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:Advanced Windows 32 Base API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		COMCTL32.dll:User Experience Controls Library:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:Crypto API32:10.0.19041.4165 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:DNS Client API DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.19041.4474 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.19041.4474 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:IP Helper API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Windows NT BASE API Client DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Windows NT BASE API Client DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		MpOav.dll:IOfficeAntiVirus Module:4.18.24050.7 (b60bcb9c79f050335eb91db16f7d3e97336ad10e-dirty):Microsoft Corporation
		NLAapi.dll:Network Location Awareness 2:10.0.19041.4123 (WinBuild.160101.0800):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		OLEAUT32.dll:OLEAUT32.DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		Ole32.dll:Microsoft OLE for Windows:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		POWRPROF.dll:Power Profile Helper DLL:10.0.19041.4165 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		Pdh.dll:Windows Performance Data Helper DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		RPCRT4.dll:Remote Procedure Call Runtime:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.19041.4165 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:Windows Shell Common Dll:10.0.19041.4165 (WinBuild.160101.0800):Microsoft Corporation
		UMPDC.dll
		USER32.dll:Multi-User Windows USER API Client DLL:10.0.19041.4165 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		WINHTTP.dll:Windows HTTP Services:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:MCI API DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:Windows Socket 2.0 32-Bit DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		Wldp.dll:Windows Lockdown Policy:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.19041.4355 (WinBuild.160101.0800):Microsoft Corporation
		bcrypt.dll:Windows Cryptographic Primitives Library:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM for Windows:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.19041.4355 (WinBuild.160101.0800):Microsoft Corporation
		extnet.dll:OpenJDK Platform binary:21.0.3.0:Microsoft
		fwpuclnt.dll:FWP/IPsec User-Mode API:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.19041.4474 (WinBuild.160101.0800):Microsoft Corporation
		java.dll:OpenJDK Platform binary:21.0.3.0:Microsoft
		javaw.exe:OpenJDK Platform binary:21.0.3.0:Microsoft
		jemalloc.dll
		jimage.dll:OpenJDK Platform binary:21.0.3.0:Microsoft
		jli.dll:OpenJDK Platform binary:21.0.3.0:Microsoft
		jna6763109378378686027.dll:JNA native library:7.0.0:Java(TM) Native Access (JNA)
		jsvml.dll:OpenJDK Platform binary:21.0.3.0:Microsoft
		jvm.dll:OpenJDK 64-Bit server VM:21.0.3.0:Microsoft
		kernel.appcore.dll:AppModel API Host:10.0.19041.3758 (WinBuild.160101.0800):Microsoft Corporation
		lwjgl.dll
		management.dll:OpenJDK Platform binary:21.0.3.0:Microsoft
		management_ext.dll:OpenJDK Platform binary:21.0.3.0:Microsoft
		msvcp140.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		msvcp_win.dll:Microsoft® C Runtime Library:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Microsoft Windows Sockets 2.0 Service Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:E-mail Naming Shim Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Windows NCrypt Router:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:OpenJDK Platform binary:21.0.3.0:Microsoft
		nio.dll:OpenJDK Platform binary:21.0.3.0:Microsoft
		ntdll.dll:NT Layer DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:Windows System Performance Objects DLL:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		pnrpnsp.dll:PNRP Name Space Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.19041.4355 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Shell Light-weight Utility Library:10.0.19041.4165 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:OpenJDK Platform binary:21.0.3.0:Microsoft
		ucrtbase.dll:Microsoft® C Runtime Library:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:21.0.3.0:Microsoft
		win32u.dll:Win32u:10.0.19041.4529 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:Microsoft WinRT Storage API:10.0.19041.4165 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.19041.3636 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:21.0.3.0:Microsoft
Stacktrace:
	at net.minecraft.client.main.Main.main(Main.java:239)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)

-- System Details --
Details:
	Minecraft Version: 1.21
	Minecraft Version ID: 1.21
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 21.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 294736416 bytes (281 MiB) / 645922816 bytes (616 MiB) up to 4294967296 bytes (4096 MiB)
	CPUs: 4
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 3 2200G with Radeon Vega Graphics    
	Identifier: AuthenticAMD Family 23 Model 17 Stepping 0
	Microarchitecture: Zen
	Frequency (GHz): 3.50
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 4
	Graphics card #0 name: AMD Radeon RX 6600
	Graphics card #0 vendor: Advanced Micro Devices, Inc.
	Graphics card #0 VRAM (MiB): 8176.00
	Graphics card #0 deviceId: VideoController1
	Graphics card #0 versionInfo: 31.0.24033.1003
	Memory slot #0 capacity (MiB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.13
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MiB): 8192.00
	Memory slot #1 clockSpeed (GHz): 2.13
	Memory slot #1 type: DDR4
	Virtual memory max (MiB): 27838.14
	Virtual memory used (MiB): 11214.51
	Swap memory total (MiB): 11543.71
	Swap memory used (MiB): 154.82
	Space in storage for jna.tmpdir (MiB): <path not set>
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): <path not set>
	Space in storage for io.netty.native.workdir (MiB): <path not set>
	Space in storage for java.io.tmpdir (MiB): available: 10574.05, total: 242112.09
	Space in storage for workdir (MiB): available: 10574.05, total: 242112.09
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx4096m
	Fabric Mods: 
		fabric-api: Fabric API 0.100.3+1.21
			fabric-api-base: Fabric API Base 0.4.42+6573ed8cd1
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.67+b5597344d1
			fabric-biome-api-v1: Fabric Biome API (v1) 13.0.28+6fc22b99d1
			fabric-block-api-v1: Fabric Block API (v1) 1.0.22+0af3f5a7d1
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.10+6573ed8cd1
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.52+0af3f5a7d1
			fabric-client-tags-api-v1: Fabric Client Tags 1.1.15+6573ed8cd1
			fabric-command-api-v1: Fabric Command API (v1) 1.2.48+f71b366fd1
			fabric-command-api-v2: Fabric Command API (v2) 2.2.27+6a6dfa19d1
			fabric-commands-v0: Fabric Commands (v0) 0.2.65+df3654b3d1
			fabric-content-registries-v0: Fabric Content Registries (v0) 8.0.13+b5597344d1
			fabric-convention-tags-v1: Fabric Convention Tags 2.0.14+7f945d5bd1
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.3.1+8a3b5617d1
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.29+0af3f5a7d1
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.1.23+6a6dfa19d1
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 20.2.8+16c4ae25d1
			fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.0+6fc22b99d1
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.6.12+6fc22b99d1
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.7.10+e633f883d1
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.52+6573ed8cd1
			fabric-item-api-v1: Fabric Item API (v1) 11.0.0+afdfc921d1
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.1.1+cb5ced13d1
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.47+0af3f5a7d1
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.45+df3654b3d1
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.3.11+8f3583aed1
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.10+6573ed8cd1
			fabric-message-api-v1: Fabric Message API (v1) 6.0.13+6573ed8cd1
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 2.0.0+fe474d6bd1
			fabric-networking-api-v1: Fabric Networking API (v1) 4.2.0+ab7edbacd1
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 15.1.11+d1321076d1
			fabric-particles-v1: Fabric Particles (v1) 4.0.2+6573ed8cd1
			fabric-recipe-api-v1: Fabric Recipe API (v1) 5.0.9+6573ed8cd1
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 5.0.22+ab7edbacd1
			fabric-renderer-api-v1: Fabric Renderer API (v1) 3.2.16+fe474d6bd1
			fabric-renderer-indigo: Fabric Renderer - Indigo 1.6.4+48fb1586d1
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.66+df3654b3d1
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.48+73761d2ed1
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.6+b5597344d1
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.69+df3654b3d1
			fabric-rendering-v1: Fabric Rendering (v1) 5.0.3+d1321076d1
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 4.2.1+d153f344d1
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 1.1.4+cb5ced13d1
			fabric-screen-api-v1: Fabric Screen API (v1) 2.0.24+b5597344d1
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.79+b5597344d1
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.23+6573ed8cd1
			fabric-transfer-api-v1: Fabric Transfer API (v1) 5.1.14+b5597344d1
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.0.12+6573ed8cd1
		fabricloader: Fabric Loader 0.15.11
			mixinextras: MixinExtras 0.3.5
		geckolib: GeckoLib 4 4.5.5
		java: OpenJDK 64-Bit Server VM 21
		minecraft: Minecraft 1.21
		netherdepthsupgrade: Nether Depths Upgrade fabric-3.1.6-1.21
	Launched Version: 1.21
	Launcher name: PrismLauncher
	Backend library: LWJGL version 3.3.3-snapshot
	Backend API: Unknown
	Window size: <not initialized>
	GFLW Platform: <error>
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: <disabled>
	Is Modded: Definitely; Client brand changed to 'fabric'
	Universe: 404
	Type: Client (map_client.txt)
	Locale: en_AU
	System encoding: Cp1252
	File encoding: UTF-8
	CPU: <unknown>
``` 

</details> 

## Log

[latest.log](https://github.com/user-attachments/files/15936525/latest.log)

<details><summary>Click to expand log</summary>

```
[21:49:11] [main/INFO]: Loading Minecraft 1.21 with Fabric Loader 0.15.11
[21:49:11] [main/INFO]: Fabric is preparing JARs on first launch, this may take a few seconds...
[21:49:11] [ForkJoinPool-1-worker-4/WARN]: Mod netherdepthsupgrade uses the version fabric-3.1.6-1.21 which isn't compatible with Loader's extended semantic version format (Could not parse version number component 'fabric'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[21:49:11] [main/INFO]: Loading 54 mods:
	- fabric-api 0.100.3+1.21
	   |-- fabric-api-base 0.4.42+6573ed8cd1
	   |-- fabric-api-lookup-api-v1 1.6.67+b5597344d1
	   |-- fabric-biome-api-v1 13.0.28+6fc22b99d1
	   |-- fabric-block-api-v1 1.0.22+0af3f5a7d1
	   |-- fabric-block-view-api-v2 1.0.10+6573ed8cd1
	   |-- fabric-blockrenderlayer-v1 1.1.52+0af3f5a7d1
	   |-- fabric-client-tags-api-v1 1.1.15+6573ed8cd1
	   |-- fabric-command-api-v1 1.2.48+f71b366fd1
	   |-- fabric-command-api-v2 2.2.27+6a6dfa19d1
	   |-- fabric-commands-v0 0.2.65+df3654b3d1
	   |-- fabric-content-registries-v0 8.0.13+b5597344d1
	   |-- fabric-convention-tags-v1 2.0.14+7f945d5bd1
	   |-- fabric-convention-tags-v2 2.3.1+8a3b5617d1
	   |-- fabric-crash-report-info-v1 0.2.29+0af3f5a7d1
	   |-- fabric-data-attachment-api-v1 1.1.23+6a6dfa19d1
	   |-- fabric-data-generation-api-v1 20.2.8+16c4ae25d1
	   |-- fabric-dimensions-v1 4.0.0+6fc22b99d1
	   |-- fabric-entity-events-v1 1.6.12+6fc22b99d1
	   |-- fabric-events-interaction-v0 0.7.10+e633f883d1
	   |-- fabric-game-rule-api-v1 1.0.52+6573ed8cd1
	   |-- fabric-item-api-v1 11.0.0+afdfc921d1
	   |-- fabric-item-group-api-v1 4.1.1+cb5ced13d1
	   |-- fabric-key-binding-api-v1 1.0.47+0af3f5a7d1
	   |-- fabric-keybindings-v0 0.2.45+df3654b3d1
	   |-- fabric-lifecycle-events-v1 2.3.11+8f3583aed1
	   |-- fabric-loot-api-v2 3.0.10+6573ed8cd1
	   |-- fabric-message-api-v1 6.0.13+6573ed8cd1
	   |-- fabric-model-loading-api-v1 2.0.0+fe474d6bd1
	   |-- fabric-networking-api-v1 4.2.0+ab7edbacd1
	   |-- fabric-object-builder-api-v1 15.1.11+d1321076d1
	   |-- fabric-particles-v1 4.0.2+6573ed8cd1
	   |-- fabric-recipe-api-v1 5.0.9+6573ed8cd1
	   |-- fabric-registry-sync-v0 5.0.22+ab7edbacd1
	   |-- fabric-renderer-api-v1 3.2.16+fe474d6bd1
	   |-- fabric-renderer-indigo 1.6.4+48fb1586d1
	   |-- fabric-renderer-registries-v1 3.2.66+df3654b3d1
	   |-- fabric-rendering-data-attachment-v1 0.3.48+73761d2ed1
	   |-- fabric-rendering-fluids-v1 3.1.6+b5597344d1
	   |-- fabric-rendering-v0 1.1.69+df3654b3d1
	   |-- fabric-rendering-v1 5.0.3+d1321076d1
	   |-- fabric-resource-conditions-api-v1 4.2.1+d153f344d1
	   |-- fabric-resource-loader-v0 1.1.4+cb5ced13d1
	   |-- fabric-screen-api-v1 2.0.24+b5597344d1
	   |-- fabric-screen-handler-api-v1 1.3.79+b5597344d1
	   |-- fabric-sound-api-v1 1.0.23+6573ed8cd1
	   |-- fabric-transfer-api-v1 5.1.14+b5597344d1
	   \-- fabric-transitive-access-wideners-v1 6.0.12+6573ed8cd1
	- fabricloader 0.15.11
	   \-- mixinextras 0.3.5
	- geckolib 4.5.5
	- java 21
	- minecraft 1.21
	- netherdepthsupgrade fabric-3.1.6-1.21
[21:49:13] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/C:/Users/Anon/AppData/Roaming/PrismLauncher/libraries/net/fabricmc/sponge-mixin/0.13.3+mixin.0.8.5/sponge-mixin-0.13.3+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[21:49:16] [main/INFO]: Compatibility level set to JAVA_17
[21:49:19] [main/INFO]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.3.5).
[21:49:22] [Datafixer Bootstrap/INFO]: 226 Datafixer optimizations took 901 milliseconds
[21:49:22] [main/WARN]: Invalid registry value type detected for PerfOS counters. Should be REG_DWORD. Ignoring: HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\PerfOS\Performance\Disable Performance Counters.
[21:49:34] [Render thread/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[21:49:34] [Render thread/INFO]: Setting user: Anon
``` 

</details>

**Comment by Scouter456** [comment]
Should be fixed in latest update

**Comment by Poopooracoocoo** [comment]
I confirm that this has been fixed with Nether Depths Upgrade fabric-3.1.7-1.21.
